internal/connect/command_custom_plugin_create.go

```diff
@@ -66,6 +68,10 @@ func (c *customPluginCommand) createCustomPlugin(cmd *cobra.Command, args []stri
 	if err != nil {
 		return err
 	}
+	cloud, err := cmd.Flags().GetString("cloud")
+	if err != nil {
```

Comment:
this introduces breaking change, we should avoid. 

cloud is optional in api level. we should carried over same behavior  to the cli as well. aka, allow empty cloud attribute when create plugin.

Reply:
It also looks like it's optional here, no? Not sure what the breaking change is.

Reply:
yes, it is optional when create plugin too.

Reply:
We want the cloud input be optional. My understanding is `cobra.CheckErr(cmd.MarkFlagRequired("connector-type"))` will check the flag is required or not. So for cloud we don't check it.  This looks like it's optional here